demos/continuous_batching/accuracy/README.md

```diff
@@ -107,7 +103,33 @@ openai_compatible (model_version=OpenGVLab/InternVL2_5-8B,max_retries=1), gen_kw
 ```
 
 
+## Running the tests for agentic models with function calls
+
+Use [Berkeley function call leaderboard ](https://github.com/ShishirPatil/gorilla/tree/main/berkeley-function-call-leaderboard)
+
+
+```bash
+git clone https://github.com/ShishirPatil/gorilla
+cd gorilla/berkeley-function-call-leaderboard
+git checkout ac37049f00022af54cc44b6aa0cad4402c22d1a0
+git apply gorila.patch
+pip install -e . 
+export OPENAI_BASE_URL=http://localhost:8000/v3
+bfcl generate --model openvino-qwen3-8b-int8-FC --test-category multiple --num-threads 100 -o
+bfcl evaluate --model openvino-qwen3-8b-int8-FC
+```
+
+**Analyzing results**
+The command above assumes the models is deployed with the name `openivno-qwen3-8b-int8`. It must match the name set in the `bfcl_eval/constants/model_config.py`.
```

Comment:
The part with explaining how the model should be deployed could be placed before `bfcl generate` since user needs correctly deployed OVMS for this command to succeed.